src/scss/constants.scss

```diff
@@ -81,6 +86,7 @@ $z-layers: (
   navbar-schedule-backdrop: $z-layer-live-schedule,
   navbar-schedule: $z-layer-live-schedule,
   navbar: $z-layer-navbar,
+  navbar__search-input: 2,
```

Comment:
Are there any `navbar-*` entries here that are no longer used in the codebase (eg. `navbar-nav-search-results`, etc)? If so, we should remove them from this helper too.

Reply:
removed `navbar-nav-search-results` and `navbar__search-input` : https://github.com/sparkletown/sparkle/pull/1328/commits/16a2051fbc83a252406d90106ce1cc345aecf83a